### PR TITLE
Update ProxyType for runtimes

### PR DESCRIPTION
**Pull request by lemunozm**
# Description

Related to #1461 

This PR adds the missing loans extrinsic to `ProxyType` for all runtimes.

I've seen that `ProxyType::NonTransfer` for centrifuge runtime is not configured with the pool/loan/permission pallets. Is this deliberate, or did we miss this from the last release? @branan @wischli

### CODE REVIEW

**Comment by mikiquantum** [comment]
I will merge up once I get some approvals from @branan and/or @wischli . I think I fixed the doc gh action in #1462

**Comment by branan** [comment]
> I've seen that ProxyType::NonTransfer for centrifuge runtime is not configured with the pool/loan/permission pallets. Is this deliberate, or did we miss this from the last release?

Probably an oversight, since the UI & expected workflows around Pools would never use that proxy type. I at least don't recall any discussion where we decided to leave them out intentionally.

I think it would really just be a consistency fix, since we wouldn't expect any users to want to set up a proxy type like that to do pools things in production.

**Review by branan — DISMISSED** [review]
LGTM, hopefully a rebase picks up the docs job fix and this goes green.

**Comment by lemunozm** [comment]
Double-check @branan to lines added to `centrifuge`. I've replicated what I found in `altair`. The PR should be ready now to be merged.

**Review by branan — APPROVED** [review]

**Pull request merged**
